src/sdk/cpp/lib/include/call.hpp

```diff
@@ -315,20 +318,15 @@ class PreparedFunctionReply {
      *
      * @param talent_id The ID of the talent to send the reply to
      * @param feature The name of the feature
-     * @param subject The subject of the original event resulting in the
-     * function call
-     * @param channel_id The ID of the calling talent's channel_id
-     * @param call_id The unique call IDs
+     * @param event The event that triggered the call
      * @param return_topic The topic on which the reply should be posted
-     * @param publisher A pointer to a Publisher
+     * @param gatewy A pointer to a GatewayProtocl
```

Comment:
typo gatew(a)y

Reply:
I'm going to update the documentation and provide unit tests in a different commit. If it's ok with you I'd like to merge this and correct it in the next PR.